libs/sdk-utils/multicall/src/lib/sdk-utils-multicall.ts

```diff
@@ -1,23 +1,70 @@
-export function sdkUtilsMulticall(): string {
-  return 'sdk-utils-multicall';
-}
-
+import { ContractWrapper, decorate } from '@sdk-utils/core';
+import { providers as _0xS } from '@0xsequence/multicall';
+import { Contract, ethers } from 'ethers';
+import { Provider } from '@ethersproject/providers';
+import { MulticallProvider } from '@0xsequence/multicall/dist/declarations/src/providers';
 
 /**
- * MyType extends a `thing` that does some `stuff`
+ * Multicall implementation with 0xSequence happens automatically.
+ * There are no special methods to call, but we add this flag purely as an indicator to show the contract has been wrapped.
  */
-interface MyType {
-  // the value of choice
-  myValue: string
-}
+type Multicall = {
+  isMulticallEnabled: boolean;
```

Comment:
What's the purpose of this flag? I'm assuming that if something has the type `Multicall` it means that it is multicall-enabled.

Reply:
The multicall provider stores multicall info inside a private class, so we can't run checks at runtime. This is a very simple flag we can use for conditionals and testing. It's not strictly necessary though.

Reply:
👍